### ISSUE TITLE
Don't need login, make GenToken flexible in each request

### ISSUE THREAD

**Issue by SanhNX** [arvindr21/myRESTApp#3]
Sometime, i don't want login to get Access Token, I just want assign access token when i call "GET" method.
my case mean, with each request we need get access token and assign it to primary request to fetch data, when this request success, we set access token expire manual.

Please help me resolve this case. Thank you.

**Comment by arvindr21** [comment]
@SanhNX are you asking if you could override the expiry of the token manually?

**Comment by SanhNX** [comment]
yep, i want to override behavior of the token because your app need login first to get access token, and my case do not need login, i want to get access token anytime and assign it to each request then expiry when request is done.

The reason why I want to do because I do not want anonymous users to access directly into the path API to retrieve data
